### PR TITLE
Add a namespace selector to instaslice webhook

### PR DESCRIPTION
**Pull request by sairameshv**
Fixes https://github.com/openshift/instaslice-operator/issues/93
Updated the instaslice-operator namespace to `instaslice-system`

/cc @asm582 @harche @rphillips

### CODE REVIEW

File: config/default/namespace_selector_patch.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+apiVersion: admissionregistration.k8s.io/v1
+kind: MutatingWebhookConfiguration
+metadata:
+  name: mutating-webhook-configuration
+webhooks:
+- name: instaslice.codeflare.dev
```
Comment (kannon92) [inline config/default/namespace_selector_patch.yaml:7]: Can we drop the code flare name?

Comment (asm582) [reply config/default/namespace_selector_patch.yaml:7]: what should be the new name?

Comment (kannon92) [reply config/default/namespace_selector_patch.yaml:7]: instaslice.dev?

Comment (harche) [reply config/default/namespace_selector_patch.yaml:7]: instaslice.webhook.<purpose> ?

Comment (sairameshv) [reply config/default/namespace_selector_patch.yaml]: Updated to `instaslice.dev`

Comment (asm582) [reply config/default/namespace_selector_patch.yaml]: Thanks @sairameshv

**Review by kannon92 — COMMENTED** [review]
/lgtm

**Pull request merged**